scalyr_agent/scalyr_client.py

```diff
@@ -489,6 +518,11 @@ def __receive_response(self, body_str, send_time):
                     )
                 return "requestFailed", len(body_str), response
 
+            try:
```

Comment:
Can you add a comment why it's ok to ignore the exception.

Reply:
Here we go - 61a262fc7ff7846f2a8d7a7038460ed2cfbb9ba2.

And for completeness, here it's how it looked before this change / fix:

```bash
2020-04-30 09:38:46.303Z Level 4 [core] [scalyr_client.py:523] Response was received with body "b'{"bytesCharged":693755,"status":"success"}'"
```

And after it:

```bash
2020-04-30 09:38:46.303Z Level 4 [core] [scalyr_client.py:523] Response was received with body "{"bytesCharged":693755,"status":"success"}"
```